### ISSUE TITLE
Copy an experiment to run in a different platform

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#653]
In GitLab by @gmontanepinto on Jan 29, 2021, 11:24

I was trying to copy an existing experiment because I want to run the same configuration but in a different platform. This platform (nimbus) was not defined in the `platforms.conf` of the original experiment and I got this error when I ran the copy command (using version 3.13):

```
 [CRITICAL] On Configuration files:
Critical Issues on [Expdef] config file:
[Default] Main platform is not defined! check if [HPCARCH = nimbus5] has any typo
 [eCode=7014]
```

Maybe it would be better to just show a warning and let the copy finish, as I suppose that if the platform is not defined it will fail when the run is done. 

@dbeltrankyl do you think this could be fixed?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jan 29, 2021, 11:28

Hi @gmontanepinto ,

Yes, in fact, it was working like that not long time ago, perhaps I mess something up this week related to this while doing the fix for the dummy experiment, on Monday it will be fixed again.

I have a check to avoid these things but apparently, it is not working for that issue right now `autosubmit_config.check_conf_files(False)`

For now , copy with 3.12 and switch back to 3.13

**Comment by kinow** [comment]
In GitLab by @gmontanepinto on Jan 29, 2021, 11:32

Great! Thanks.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jan 29, 2021, 11:43

closed via commit b5254ddaee594fa75c1d0d8fb0ac90fc18ac6ff6

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jan 29, 2021, 11:44

reopened

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 17, 2021, 13:33

Not sure why I put this issue in testing,

It still was not working properly, I just fixed it and will be available on the next update 3.13.0b ( or in the final release)
